demos/form-control/input2.php

```diff
@@ -53,6 +53,61 @@
 $group->addControl('radio_read', [Form\Control\Radio::class, 'readOnly' => true], ['enum' => ['one', 'two', 'three']])->set('two');
 $group->addControl('radio_disb', [Form\Control\Radio::class, 'disabled' => true], ['enum' => ['one', 'two', 'three']])->set('two');
 
+$group = $form->addGroup('Sliders');
```

Comment:
I checkouted this PR and my feelings are the current state is too wild:
- because the UX is very different to other controls - for example the vertical space is too large and the control is too wide,
- there are too many options - for example labels above the horizontal axis/line, the atk4/ui framework aims to provide unified and robust developer experience, not 1:1 FUI port - think this way, in the future (in a decade or so), the FUI might became obsolete and we should be able to replace FUI and still support all supported options

So for now, please put this complex PR on hold and open another PR with minimal Slider impl. Just Slider class with normal/RO/disabled support and min/max value. This should be presented in `demos/form-control/input2.php` and min/max options in `demos/form-control/slider.php` and tested with Behat tests.

Reply:
Hi everyone, just as food for thought (courtesy of @abbadon1334) a minimal implementation of Fomantic-UI slider sample code:
```
                $slider = $this->add([View::class] )->addClass('ui labeled ticked small slider');

                $slider->js(true)->slider([
                    "min" => 0,
                    "max" => 45,
                    'showThumbTooltip' => 'true',
                    'tooltipConfig' => ['position' => 'top center',
                        'variation' => 'small visible blue'],
                    "onChange" => new \Atk4\Ui\Js\JsFunction(['v'], [
                        new \Atk4\Ui\Js\JsExpression($field->js()->find('input')->jsRender().".val(v)")]),
                ]);
```

Reply:
> So for now, please put this complex PR on hold and open another PR with minimal Slider impl. Just Slider class with normal/RO/disabled support and min/max value. This should be presented in `demos/form-control/input2.php` and min/max options in `demos/form-control/slider.php` and tested with Behat tests.

@rickyosser would you like to work on the first minimal implementation?